tsconfig.json

```
@@ -8,7 +8,8 @@
     "outDir": "lib/",
     "strict": false,
     "target": "es2018",
+    "declaration": true,
   },
-  "include": ["next-env.d.ts", "src/*.ts", "src/*.tsx"],
+  "include": ["next-env.d.ts", "src/*.ts", "src/*.tsx", "src/**/*.ts", "src/**/*.tsx"],
```

```suggestion
  "include": ["next-env.d.ts", "src/**/*.ts", "src/**/*.tsx"],
```
`/**` matches 0-∞ levels of directories.

---

Good catch, I thought I tried that and it didn't work but it works.